openvmm/hvlite_entry/src/cli_args.rs

```diff
@@ -766,6 +770,30 @@ impl FromStr for FloppyDiskCli {
     }
 }
 
+#[derive(Clone)]
+pub struct DebugconSerialConfigCli {
+    pub port: u16,
```

Comment:
Why wouldn't this just always be the Bochs 0xe9?

Reply:
Nope! It seems that the precise port location varies depending on who's using it. e.g: SeaBIOS uses 0x402

Reply:
Weiiiird.

Reply:
I guess it's not a very sophisticated device so perhaps it was just reinvented multiple times.

Reply:
You'd think so, and yet, the 0x402 port uses the same magic readback value as the OG bochs 0xe9 port?